src/components/modals/ConfirmationModal.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <ModalBase class="confirmation-modal" @close="onClose()">
+    <template #title>
+      <h3 class="app-form__title">Are you sure?</h3>
+    </template>
+    <template #main>
+      <div class="app-form" @submit.prevent>
+        <div class="app-form__main">
+          <div class="app-form__field">
+            {{ text }}
+          </div>
+        </div>
+        <div class="app-form__footer">
+          <button class="app-btn app-btn_outlined" @click="onClose()">
+            No
+          </button>
+          <button class="app-btn" type="button" @click="onSubmit()">Yes</button>
```

Comment:
remove (), @click event call function by itself

Reply:
if you want to use form, set type="submit" and in form tag, call @submit.prevent="submit function"